site/content/docs/main/backup-repository-configuration.md

```diff
@@ -49,6 +49,33 @@ Below is the supported configurations by Velero and the specific backup reposito
 ***Kopia repository:***  
 `cacheLimitMB`: specifies the size limit(in MB) for the local data cache. The more data is cached locally, the less data may be downloaded from the backup storage, so the better performance may be achieved. Practically, you can specify any size that is smaller than the free space so that the disk space won't run out. This parameter is for repository connection, that is, you could change it before connecting to the repository. E.g., before a backup/restore/maintenance.  
 
+### Full Maintenance Interval customization
```

Comment:
According to the existing org of this doc (options are repository specific), it's better to add this section under `Kopia repository:` together with `cacheLimitMB`

Reply:
393137c85bbea8bdd30900c57c750be8038ba335